osmomath/decimal_test.go

```diff
@@ -356,40 +382,40 @@ func (s *decimalTestSuite) TestDecsEqual() {
 func (s *decimalTestSuite) TestArithmetic() {
```

Comment:
Note to reviewer: all tests below are copied from: https://github.com/osmosis-labs/cosmos-sdk/pull/430